[RFC] doc: remove :fixdel

justinmk opened the pull request:
`:fixdel` was removed in 3baba1e7bc6698e6bc9f1d37fce88b30d6274bc9.

Possibly related: https://github.com/neovim/neovim/issues/2048

justinmk committed:
doc: remove :fixdel

:fixdel was removed in 3baba1e7bc6698e6bc9f1d37fce88b30d6274bc9

Helped-by: Michael Reed <m.reed@mykolab.com>

Pyrohh commented on runtime/doc/change.txt:
```diff
@@ -32,9 +32,8 @@ For inserting text see |insert.txt|.
-			want.  See |'whichwrap'| for deleting a line break
-			(join lines).  {Vi does not support <Del>}
+                        See |'whichwrap'| for deleting a line break (join
+                        lines).  {Vi does not support <Del>}
```
Tabs, not spaces.

justinmk replied:
:+1:

Pyrohh commented on runtime/doc/various.txt:
```diff
@@ -35,8 +35,6 @@ CTRL-L			Clear and redraw the screen.  The redraw may happen
 			Note: if you like to use <BS> for this, add this
 			mapping to your .vimrc: >
 				:map CTRL-V <BS>   CTRL-V <Del>
-<			See |:fixdel| if your <Del> key does not do what you
```
The `<` was removed, but I'm not sure it was accidental (isn't only a `>` needed for one line of code in help docs?).

justinmk replied:
:+1:

Pyrohh commented:
There's two more instances of this, just run `git grep ':fix'`.  Also this should probably be added to `vim-differences.txt`.

justinmk commented:
>  should probably be added to vim-differences.txt.

We just need a single line in there that says "any terminal fiddling was removed". The `t_Foo` hacks supported by vim aren't supported by nvim. `:fixdel` is just setting `t_Foo` codes in a different way.

Pyrohh commented:
> We just need a single line in there that says "any terminal fiddling was removed". The t_Foo hacks supported by vim aren't supported by nvim. :fixdel is just setting t_Foo codes in a different way.

I don't see a downside in just saying `:fixdel` was removed, and then saying the `t_Foo` stuff was removed on a below line. "terminal fiddling" is too vague IMO.

justinmk committed:
doc/vim_diff.txt: formatting

- backticks in Vim help docs are supported by even the default
  colorscheme; we should start using them for technical identifiers
  instead of quotation marks.
- prefer judicious indentation to gratuitous bullets

Helped-by: Michael Reed <m.reed@mykolab.com>

justinmk commented:
updated

justinmk commented on runtime/doc/vim_diff.txt:
```diff
@@ -21,9 +21,9 @@ these differences.
-- Use ".nviminfo" instead of ".viminfo" for persistent session information.
+- Use `.nvimrc` instead of `.vimrc` for storing configuration.
+- Use `.nvim` instead of `.vim` to store configuration files.
+- Use `.nviminfo` instead of `.viminfo` for persistent session information.
```
vimdoc has supported backticks for a long time, though it's odd it is not mentioned in `:help help`. We should start using these for technical identifiers instead of quotation marks, in most cases.

Pyrohh replied:
> though it's odd it is not mentioned in :help help

Since this PR's scope doesn't seem specific anymore, perhaps you could add this patch to this PR:

```diff
diff --git a/runtime/doc/helphelp.txt b/runtime/doc/helphelp.txt
index ca109cf..5e3db84 100644
--- a/runtime/doc/helphelp.txt
+++ b/runtime/doc/helphelp.txt
@@ -318,11 +318,14 @@ name between two bars (|) eg. |help-writing|.
 When referring to a Vim option in the help file, place the option name between
 two single quotes, eg. 'statusline'
 
+When referring to any other technical term, such as a filename or function
+parameter, surround it in backticks (`), e.g., `~/.path/to/nvimrc`.
+
 
 HIGHLIGHTING
 
 To define a column heading, use a tilde character at the end of the line.
-This will highlight the column heading in a different color.  E.g.
+This will highlight the column heading in a different color.
 
 Column heading~
```

Pyrohh commented on runtime/doc/vim_diff.txt:
```diff
@@ -49,27 +49,27 @@ are always available and may be used simultaneously in separate plugins.  The
+  <C-BS>, <C-S-BS>
+  <C-Enter>, <C-S-Enter>
 
 Events:
```
I thought I wouldn't like a lack of dashes, but I think it looks cleaner so :+1:

One nit though: IMO these headings should either have the colon removed to match what's done in section 5, or the headings in section 5 should have colons added.

Pyrohh commented:
Besides the comments I made, LGTM.

justinmk committed:
doc/helphelp.txt: mention backticks for technical identifiers

[ci skip]
